src/Launcher.vala

```diff
@@ -15,6 +15,7 @@ public class Dock.Launcher : Gtk.Box {
         }
     }
 
+    public signal void removed ();
```

Comment:
Do we need this signal? Can we just use `launcher.app.removed` instead?

Reply:
this was already preparation for making the launcher manager only know about a launcher base class and don't care about what it actually represents (app/workspace/background app collection)
I can remove it for now if you prefer 🤷